### PR TITLE
UI CDN

### PR DESCRIPTION
**Pull request by nick**
Currently generating a separate file in the `dist` folder, `cdn.html`, so we can test

### CODE REVIEW

File: shop/public/template-cdn.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+<script>
+  if (typeof window.openpgp !== 'undefined') {
+    openpgp.config.show_comment = false
+    openpgp.config.show_version = false
+    openpgp.initWorker({ path: 'dist/openpgp.worker.min.js' })
+  }
+  (function () {
+    const uiSrc = 'UI_SRC'
+    window.webpackPublicPath = uiSrc
+    const rnd = +new Date()
```
Comment (mikeshultz) [inline shop/public/template-cdn.html:51]: Wonder if this is necessary.  Shouldn't be the end of the world if this is cached.  Though, I don't have any strong arguments for/against here.

File: shop/public/template-cdn.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  <link rel="data-dir" href="DATA_DIR" />
+  <link rel="icon" href="FAVICON" />
+</head>
+
+<body>
+  <div id="app">
+    <div class="loading-screen"></div>
+  </div>
+</body>
+<script crossorigin src="dist/openpgp.min.js"></script>
```
Comment (mikeshultz) [inline shop/public/template-cdn.html:41]: Could point these at ui.ogn.app/ui.staging.ogn.app, as well

Comment (nick) [reply shop/public/template-cdn.html:41]: I tried that originally but web workers can't work cross-domain

Comment (mikeshultz) [reply shop/public/template-cdn.html:41]: Could you elaborate?  Not really familiar.  Is web workers functionality that openpgp uses, I guess?

Comment (nick) [reply shop/public/template-cdn.html:41]: Yea it's just for the openpgp webworker stuff, which it relies on. I haven't looked into it much yet, there might be a workaround, but this was the quickest fix for now

**Pull request merged**